source/Applications/Advanced/CaptureUndistortRGB/CaptureUndistortRGB.cpp

```diff
@@ -150,7 +165,13 @@ int main()
             use2D = true;
         }
 
-        const auto bgr = (use2D ? getImage2D(camera) : getImage3D(camera));
+        const std::chrono::microseconds exposureTime = std::chrono::microseconds{ 20000 };
```

Comment:
Consider using `const auto` for all these variables.

Reply:
changed variables in main to const auto